.github/workflows/examples-java-push.yml

```diff
@@ -55,3 +55,30 @@ jobs:
           name: examples-java-reports
           path: '**/build/reports/*'
           retention-days: 5
+      - name: Upload spring-boot manifest
+        uses: actions/upload-artifact@v2
+        with:
+          name: spring-boot-manifest
+          path: 'spring-boot/manifest.yml'
+          retention-days: 5
+  deploy:
+    environment: sandbox
+    runs-on: ubuntu-18.04
+    needs: test
+#    if: github.ref == 'refs/heads/main'
```

Comment:
```suggestion
    if: github.ref == 'refs/heads/main'
```

Reply:
just seeing if it still works with my changes, will put back again after

Reply:
NP, just so we didn't forget :upside_down_face: